eplaunch/__init__.py

```diff
@@ -0,0 +1,2 @@
+NAME = "EP-Launch"
+VERSION = "0.1"
```

Comment:
I moved a couple parameters into the core `__init__` file of the library, which is fairly common.